some updates for Skinning wiki (#107)
Quaver/Quaver.Wiki: en-US/skins/README.md

If you want to start creating skin elements from the scratch, you will need an image editing software of your choice. We made a list of the software below:

   - [Adobe Photoshop](https://www.adobe.com/products/photoshop.html?promoid=PC1PQQ5T&mv=other)
   - [GIMP](https://www.gimp.org)
   - [paint.net](https://www.getpaint.net/index.html)
   - [Figma](https://www.figma.com/login)
   - [Photopea](https://www.photopea.com)

## Where can I get skins that I saw someone played with in Quaver?

There are two ways of getting skins made by other people:

* [Quaver's Steam Workshop](https://steamcommunity.com/app/980610/workshop/)
  
  You can get skins in the Steam Workshop by clicking on a skin of your choice, and click Subscribe. After subscribing, Steam will download the skin files from the workshop. When the download is complete, the skin should be available in Skin Settings. Skins that you have currently subscribed to are in blue font color.

* Asking the person for the skin they're using.
  
  Some people do not upload their skins in the Steam Workshop. You may ask them for the skin but they may not give it to you.

## Can I make my own edition of someone else's skin?

> Yes, you can make your own edition or use skin elements from other skins and you are allowed to upload them in the Steam Workshop.

Reviewer: ```suggestion
Yes, you can make your own edits or use skin elements from other skins, and you are allowed to upload them in the Steam Workshop if the original creator(s) allow it.
```

> However, we suggest giving credit to the original author of the skin if you would use their skin elements.

Reviewer: Remove this. Users need explicit permission in order to upload another's skin elements.

## How do I reload my skin in Quaver?

> With the skin of your choice, reloading the skin is by pressing CTRL + S in the song selection menu or the main menu.

Reviewer: ```suggestion
You can reload your skin by pressing CTRL + S in the menu screens.
```

# Uploading a Skin to the Workshop

> If you want to upload your skin to the workshop, you must abide by [the rules and guidelines](https://support.steampowered.com/kb_article.php?ref=4045-USHJ-3810). Otherwise, your skin will get reported and taken down or will get blocked by the Steam Community Moderators from uploading your skins in the workshop.

Reviewer: ```suggestion
If you would like to upload your skin to the workshop, you must abide by [the rules and guidelines](https://support.steampowered.com/kb_article.php?ref=4045-USHJ-3810). Otherwise, you may not be allowed to upload it.
```

> The requirements of uploading in Steam Workshop are the following to be able to upload your skin:

Reviewer: ```suggestion
The requirements of uploading your skin to the Steam Workshop are as follows:
```

> - Your skin folder must contain a file named *steam_workshop_preview.png* (file size must be under 1MB). The file will be used as the thumbnail of your skin.

Reviewer: ```suggestion
- Your skin folder must contain a file named `steam_workshop_preview.png` (file size must be under 1MB). The file will be used as the thumbnail of your skin.
```

- Your Steam account must be eligible for uploading. If you have issues uploading your skin to the workshop, [this article](https://support.steampowered.com/kb_article.php?ref=3330-iagk-7663) may be one of the reasons.
- You must have agreed on the [Steam Subscriber Agreement](https://store.steampowered.com/subscriber_agreement/).

> We suggest putting the gameplay of your skin in the *steam_workshop_preview.png* so the players can see what the skin would and the file size should be under 1MB regardless of the image's resolution.

Reviewer: ```suggestion
We suggest having an image of gameplay for the `steam_workshop_preview.png` image, so players can see what the skin looks like prior to downloading it.
```
